app/static/css/maps.css

```diff
@@ -446,12 +446,23 @@ button {
     line-height: 18pt;
 }
 
-#footer-maps-logos-eu-ger img{
-    text-align: left;
-    height: 7rem;
+#footer-maps-logos-eu-ger,
+#footer-maps-logos-giz {
+  text-align: center;
+}
+
+#footer-maps .footer-maps__text {
+  text-align: center;
+}
+
+#footer-maps .footer-maps__text p {
+  font-size: 1rem;
+  padding-left: 0;
 }
 
-#footer-maps-logos-giz img{
-    text-align: right;
-    height: 4.09rem;
+#footer-maps-logos-eu-ger img,
+#footer-maps-logos-giz img {

```

Comment:
The height ratio between the german cooperation and the giz logo has to be very specific. I can get what is the constraint for a given width if you need. Right now I have the ratio for a given height

Reply:
I used the ratio you mentioned in #177

Reply:
I took a printscreen and checked: it respected the ratio :)